packages/relottie-metadata/README.md

```diff
@@ -0,0 +1,127 @@
+# relottie-metadata
+
+![npm](https://img.shields.io/npm/v/@lottiefiles/relottie-metadata)
+![npm](https://img.shields.io/npm/dt/%40lottiefiles/relottie-metadata)
+![npm](https://img.shields.io/npm/l/@lottiefiles/relottie-metadata)
+
+A [relottie] plugin to extract Lottie metadata from the document and store them in vfile.data.
+
+## Use
+
+```sh
+yarn add @lottiefiles/relottie-metadata
+```
+
+```ts
+import relottieParse, { type ParseFileData } from '@lottiefiles/relottie-parse';
+import relottieStringify, { type StringifyFileData } from '@lottiefiles/relottie-stringify';
+import relottieMetadata, { type MetadataFileData } from '@lottiefiles/relottie-metadata';
+import { unified } from 'unified';
+
+type FileData = ParseFileData & StringifyFileData & MetadataFileData
+
+// The Lottie source: __fixtures__/features/auto-orient.json
+const lottieJson = '...';
+
+const vfile = unified()
+  .use(relottieParse)
+  .use(relottieMetadata)
+  .use(relottieStringify)
```

Comment:
[Question]
the `relottieStringify` plugin is optional in this use case i assume ?

Reply:
you cannot call `.process` or `.processSync` without a Compiler